lightly/data/_image.py

```diff
@@ -41,16 +41,15 @@ def _is_valid_file(filename):
         _is_valid_file = is_valid_file
 
     instances = []
-    for fname in os.listdir(directory):
+    for f in os.scandir(directory):
 
-        if not _is_valid_file(fname):
+        if not _is_valid_file(f.name):
             continue
 
-        path = os.path.join(directory, fname)
-        item = (path, 0)
+        item = (f.path, 0)
```

Comment:
I know this did not change with this PR, but could you specify why the target_i is always 0?
Furthermore, I would love to see the return type of this function specified as `List[str, int]`